lib/src/sass/_modals.scss

```diff
@@ -0,0 +1,59 @@
+@import './colors';
+@import './mixins';
+
+$modal-padding: 20px;
+$header-footer-height: 75px;
+$modal-border: 1px solid $gray-200;
+$body-height: 100%;
+
+.hc-modal-header {
```

Comment:
Can you add `font-weight: 600` to your .hc-modal-header style.  We want that header text to be semi-bold.

Reply:
Done